packages/live-preview-sdk/src/liveUpdates.ts

```diff
@@ -142,17 +143,22 @@ export class LiveUpdates {
 
     // Tell the editor that there is a subscription
     // It's possible that the `type` is not 100% accurate as we don't know how it will be merged in the future.
-    this.sendMessage(LivePreviewPostMessageMethods.SUBSCRIBED, {
-      action: LivePreviewPostMessageMethods.SUBSCRIBED,
+    const message: Omit<SubscribedMessage, 'action'> | UnsubscribedMessage = {
       type: isGQL ? 'GQL' : 'REST',
       locale,
       entryId: sysId,
       event: 'edit',
       id,
       config: stringify(config),
+    };
+
+    this.sendMessage(LivePreviewPostMessageMethods.SUBSCRIBED, {
+      action: LivePreviewPostMessageMethods.SUBSCRIBED,
+      ...message,
     } as SubscribedMessage);
 
     return () => {
+      this.sendMessage(LivePreviewPostMessageMethods.UNSUBSCRIBED, message);
```

Comment:
We can then pick this message up on the user_interface and remove it from the subscription map there